### PR TITLE
stable/airflow: added horizontal pod autoscaler for Celery workers

### PR DESCRIPTION
**Pull request by mshalak-nix**
Added horizontal pod autoscaler for Celery workers. Also added possibility to set "resources" for git-sync container, to be able to use hpa with pods it is attached to.

Signed-off-by: Maksym Shalak <mshalak@n-ix.com>

#### What this PR does / why we need it:
Right now we are not able to scale Celery workers dynamically, which is a limitation in case we have hundreds of tasks at some point and 0 at other. Using HPA will allow Airflow to use resources better and process more tasks when required.

#### Which issue this PR fixes
n/a

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/airflow/README.md [hunk 1]
```diff
@@ -159,6 +159,15 @@ requests the logs from each workers individually.
 This requires to expose a port (8793) and ensure the pod DNS is accessible to the web server pod,
 which is why StatefulSet is for.
 
+#### Worker autoscaler
+Celery workers can be scaled using the Horizontal Pod Autoscaler.
+To enable it you need to set `workers.autoscaling.enabled=true` and provide `workers.autoscaling.maxReplicas`.
+`workers.replicas` value will be used as a minimum amount. 
+Provide custom metrics for autoscaler in `workers.autoscaling.metrics`, for more information see the
+[Kubernetes documentation](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis).
```
Comment (gsemet) [inline stable/airflow/README.md:167]: I would be so interested in having a real life example

**Review by gsemet — COMMENTED** [review]
I am interested in how to make this autoscaler work, which metrics can be used and how to implement it in real ! Maybe a few examples in the README?

File: stable/airflow/README.md (hunk 1 above)
Comment (mshalak-nix) [reply stable/airflow/README.md:167]: Added an example how it can be used with a memory metric.

Comment (gsemet) [reply stable/airflow/README.md:167]: amazing !

**Pull request merged**